animations/lib/src/basics/08_hero_animation.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+
+class HeroAnimationDemo extends StatelessWidget {
+  static const String routeName = '/basics/hero_animation';
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Colors.lightBlue,
+      appBar: AppBar(),
+      body: GestureDetector(
+        child: Hero(
+          tag: 'hero-page-child',
+          child: Container(
+            height: 50.0,
+            width: 50.0,
+            margin: EdgeInsets.all(10.0),
+            padding: EdgeInsets.all(10.0),
+            decoration:
+                BoxDecoration(shape: BoxShape.circle, color: Colors.white),
+            child: FlutterLogo(),
+          ),
+        ),
+        onTap: () => Navigator.of(context)
+            .push<void>(MaterialPageRoute(builder: (context) => HeroPage())),
+      ),
+    );
+  }
+}
+
+class HeroPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Colors.lightBlue,
+      appBar: AppBar(),
+      body: Center(
+        child: Hero(
+          tag: 'hero-page-child',
+          child: Container(
+            height: 100.0,
+            width: 100.0,
+            padding: EdgeInsets.all(10.0),
+            decoration:
+                BoxDecoration(shape: BoxShape.circle, color: Colors.white),
+            child: FlutterLogo(),
+          ),
```

Comment:
This Container could be abstracted out into a StatelessWidget, and used in both HeroAnimationDemo and HeroPage, reducing overall code length.